pkg/api/validation/validation.go

```diff
@@ -35,8 +28,18 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/util/validation"
 	"k8s.io/kubernetes/pkg/util/validation/field"
+	"math"
```

Comment:
What is going on here - do you have a different version of goimports or something?